pkg/tfbridge/auto_aliasing.go

```diff
@@ -16,6 +16,8 @@ package tfbridge
 
 import (
 	"github.com/Masterminds/semver"
+	"runtime/debug"
+	"strings"
```

Comment:
Nit: Our style has std lib modules in a different block then foreign modules.
```suggestion
	"runtime/debug"
	"strings"
	
	"github.com/Masterminds/semver"
```